misk/src/main/kotlin/misk/healthchecks/HealthCheck.kt

```diff
@@ -0,0 +1,21 @@
+package misk.healthchecks
+
+import misk.web.actions.ReadinessCheckAction
+
+/**
+ * Allows users to define custom health checks. An app with a failing HealthCheck will fail the
+ * readiness check in [ReadinessCheckAction], indicating that the app should not accept traffic.
+ */
+interface HealthCheck {
+    /**
+     * Computes whether a component of an application is healthy. For example, an implementing class
+     * can check database connectivity. As this method may involve a network call, callers should
```

Comment:
Remove the “As this method...” sentence; it’s prescriptive in the wrong way.

Reply:
Removed.